client/src/components/UI_UserInfo.re

```diff
@@ -1,46 +1,42 @@
 open GqlUserInfo;
-
 open Utils;
 
-let component = ReasonReact.statelessComponent("UI_UserInfo");
-let make = (~userId, _children) => {
-  ...component,
-  render: _ => {
-    let query = UserInfoGql.make(~userId, ());
-    <UserInfoComponent variables=query##variables>
-      ...{({result}) =>
-        switch (result) {
-        | Loading => <div> {ReasonReact.string("Loading")} </div>
-        | Error(error) => <div> {ReasonReact.string(error##message)} </div>
-        | Data(response) =>
-          response##user
-          ->(
-              arrayFirst(
-                ~empty=("Can't find user with id" ++ userId)->str, ~render=user =>
-                <dl>
-                  {user##name
-                   =>> (
-                     name => <> <dt> "Name"->str </dt> <dd> name->str </dd> </>
-                   )}
-                  <>
-                    <dt> "Username"->str </dt>
-                    <dd> {user##username->str} </dd>
-                  </>
-                  {user##avatar
-                   =>> (
-                     avatar =>
-                       <img
-                         src=avatar
-                         width="80"
-                         height="80"
-                         alt={user##username ++ " avatar"}
-                       />
-                   )}
-                </dl>
-              )
+[@react.component]
+let make = (~userId) => {
+  let query = UserInfoGql.make(~userId, ());
+  <UserInfoComponent variables=query##variables>
+    ...{({result}) =>
```

Comment:
I don't think spread operator `...` is needed with jsx3. I am pretty sure this won't compile.

Reply:
Removed